samples/RXPTest/src/Tests/DragAndDropTest.tsx

```diff
@@ -185,55 +214,101 @@ class DragAndDropView extends RX.Component<RX.CommonProps, DragAndDropViewState>
         );
     }
 
-    private _onDragEnterForFiles = (e: RX.Types.DragEvent) => {
+    private _getFilesDropped(e: RX.Types.DragEvent) {
```

Comment:
Seems `.map` is more appropriate to use in this case.

```typescript
private _getFilesDropped(e: RX.Types.DragEvent): FileInfo[] {
    return _.map(e.dataTransfer.files, (file) => ({
        fileName: file.name,
        fileSize: Math.round(fileData.size / 1024),
    }));
}
```

What do you think? ;)